schema/overview.md

```diff
@@ -0,0 +1,68 @@
+IPLD Schemas Overview
+=====================
+
+
+Goals
+-----
+
+1. Provide a reasonable, easy to use, general type system for declaring
```

Comment:
the relationship between "type system" and "schemas" seems to be in debate at the moment, we need to have a clear resolution of these two ideas before this lands

Reply:
I've been using the two more or less interchangeably.  "IPLD Schemas" are the definite noun (we're *not* calling this the proper-noun "IPLD type system")... but they're a "type system" of some general sort.

I think we hit a "limits of language" thing somewhere in this area, tbh.  Lots of this system's designs are coming from "type theory" (for programming languages) -- so the term seems to apply -- even though we're explicitly bending and limiting it to working over data and *not* function signatures (which drastically simplifies and slightly pivots the domain) -- and that puts us in a space that's... not... super well explored or well defined, relatively speaking, in popular literature.

So I'm generally +1 to trying to clarify that, but with the caveat of I think we'll absolutely land in "what are words" territory at some point and have to call it good enough.